tests/test_cosalib_cmdlib.py

```diff
@@ -151,3 +151,33 @@ def test_image_info(tmpdir):
         '-o', 'force_size,subformat=fixed',
         f"{tmpdir}/test.vpc", "10M"])
     assert cmdlib.image_info(f"{tmpdir}/test.vpc").get('format') == "vpc"
+
+
+def test_merge_dicts(tmpdir):
+    x = {
+        1: "Nope",
+        2: ["a", "b", "c"],
+        3: {"3a": True}
+    }
+
+    y = {4: True}
+
+    z = {1: "yup",
+         3: {
+             "3a": False,
+             "3b": "found"
+            }
+    }
+
+    m = cmdlib.merge_dicts(x, y)
+    for i in range(1, 4):
+        assert i in m
+
+    m = cmdlib.merge_dicts(x, z)
+    assert m[1] != "yup"
```

Comment:
I think it's better to directly check this is `Nope` rather than just that it's not `yup`.

Reply:
Updated :)